File: src/laboratorio_programacion/clase_09_06/CasoUno.py

if __name__ == '__main__':

    nota = int(input(f"Ingrese calificación del examen (1-10): "))

    if (nota >= 8) :
        print("Promocionado")
    elif (nota >= 6) :
            print("Regular")
    else :
         print("Desaprobado / Recuperatorio")

File: src/laboratorio_programacion/clase_09_06/casoUno.c
#include <stdio.h>

int main()
{
    int nota;
    printf("Ingrese calificación del examen (1-10): ");
    scanf("%d", &nota);

    if (nota >= 8)
    {
        printf("Promocionado\n");
    }
    else if (nota >= 6)
    {
        printf("Regular\n");
    }
    else
    {
        printf("Desaprobado / Recuperatorio\n");
    }
    return 0;
}